src/components/Tree/index.js

```diff
@@ -37,24 +57,36 @@ Tree.propTypes = {
         }),
     ),
     /** The action triggered when the user clicks in the tree node expand or collapse button. */
-    onExpandCollapse: PropTypes.func,
+    onNodeExpand: PropTypes.func,
     /** The action triggered when the user clicks in the tree node checkbox. */
-    onSelect: PropTypes.func,
+    onNodeCheck: PropTypes.func,
+    /** The action triggered when the user clicks in the tree node label. */
+    onNodeSelect: PropTypes.func,
+    /** The tree node name. */
+    selectedNode: PropTypes.string,
     /** A CSS class for the outer element, in addition to the component's base classes. */
     className: PropTypes.string,
     /** An object with custom style applied for the outer element. */
     style: PropTypes.object,
     /** The id of the outer element. */
     id: PropTypes.string,
+    /** The label of node with role="tree". */
```

Comment:
we need to explain more here, also role="tree" is an implementation detail and they don't know who has this role